app/src/pages/inside/logsPage/historyLine/historyLineItem/historyLineItemTooltip/historyLineItemTooltip.scss

```diff
@@ -57,8 +57,21 @@
   }
 }
 
+.launch-name-value {
+  width: 279px;
+  font-size: 13px;
+  font-weight: 600;
+  font-family: $FONT-SEMIBOLD;
+  word-wrap: break-word;
+  display: -webkit-box;
```

Comment:
`display: -webkit-box;` doesn't work in IE. I suggest using `@supports` at-rule in Sass to introduce fallback for IE and any other browsers.